pyproject.toml

```diff
@@ -33,11 +32,14 @@ pinecone-text = "^0.7.1"
 tokenizers = "^0.15.0"
 transformers = "^4.35.2"
 sentencepiece = "^0.1.99"
+pandas = "2.0.0"
+pyarrow = "^14.0.1"
```

Comment:
Why do we need these? Are they coming from the client?  
Is there any code in Canopy that directly imports `pandas` or `pyarrow`? If not I think we can remove from here, and rely on the Client's dependencies indirectly.

Reply:
I think these were actually coming from `pinecone-datasets` which we removed anyway. Not sure we need them anymore

Reply:
We need both the dependencies. It should have been a requirement in the first place. So after removing datasets it started to shout so I added them.